tests/transforms/test_experimental/test_transform_dispatcher.py

```diff
@@ -265,6 +265,30 @@ def qnode_circuit(a):
         assert transform_container.classical_cotransform is None
         assert not expand_transform_container.is_informative
 
+    def test_transform_add_to_program(self):
+        """Test to add a transform with expand transform to a program."""
+
+        dispatched_transform = transform(first_valid_transform, expand_transform=expand_transform)
+
+        program = qml.transforms.core.TransformProgram()
+        program_transformed = dispatched_transform.add_to_program(program, 0)
```

Comment:
adding to above, I'd actually prefer you _don't_ return it, because if I saw only this I might assume that `program` has not been modified, but it has (right?)

if there's an implicit copy I'm not seeing somewhere, can you just assert that `program is not program_transformed` and that it hasn't been changed?